networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc

```diff
@@ -8,6 +8,11 @@ toc::[]
 
 As a cluster administrator, you can rollback to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin if the migration to OpenShift SDN is unsuccessful.
 
+[IMPORTANT]
+====
+On {vmw-full}, if you upgraded your {product-title} 4.12 cluster to {product-title} {product-version} and you want your upgraded cluster to use the OVN-Kubernetes network plugin, you might experience a node VIP address issue with the Cloud Controller Manager. This issue impacts the ability of the Cloud Controller Manager to pull `quay.io` images into your cluster. As a workaround for this issue, specify the shared VIP address in the `networkSubnetCidr` parameter of your cluster configuration.
+====
+
```

Comment:
The `failed to get node address from cloud provider that matches ip` error is also seen on OpenShiftSDN, so I don't think we need a note for rollback.